content/blog/2017/08/2017-08-23-pull-requests-and-more.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Jenkins Needs You - Pull Request Corner at Jenkins World 2017"
+tags:
+- event
+- jenkinsworld
+author: markewaite
+---
+
+[NOTE]
+====
+This is a guest post by link:https://github.com/markewaite[Mark Waite], who maintains
+the link:https://plugins.jenkins.io/git[git plugin],
+the link:https://plugins.jenkins.io/git-client[git client plugin],
+and is a technical evangelist for link:http://cloudbees.com[CloudBees, Inc.]
+====
+
+The Jenkins project booth at link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2017]
+will include the "Pull Requests Corner", recruiting new Jenkins contributors.
+We think there are many people who will attend the conference without realizing how easy it is
+to help the Jenkins project, and how much the help is appreciated.
+
+Meet us in the "Pull Requests Corner" and we'll help you find a way to help Jenkins.
+Here are some areas where we can use your help.
+Most of them do not require coding, and do not require a large time commitment.
+
+=== One Minute Feedback on Your Version
+
+The Jenkins changelog pages (link:https://jenkins.io/changelog-stable/[LTS] and link:https://jenkins.io/changelog-stable/[weekly]) gather user experiences with specific Jenkins versions.
+You can help other Jenkins users by clicking one of the weather icons in the link:https://jenkins.io/changelog-stable/[LTS changelog] (or the link:https://jenkins.io/changelog-stable/[weekly changelog]) for the release you're using.
+Changelog feedback from weekly releases helps the release team select the long term support version.
+Changelog feedback from LTS releases helps other users prepare to upgrade.
+
+It takes less than a minute, and helps the community (which will ultimately help you).
+
+=== Five Minutes to Answer a Question
+
+image:/images/logos/needs-you/Jenkins_Needs_You-02.png["Jenkins needs YOU", role="right", width=480] 
+In five minutes or less, you can help other Jenkins users.
+
+For example:
+
+* Answer a question on the #jenkins channel of link:http://webchat.freenode.net/?channels=jenkins[internet relay chat (IRC)]
+* Answer a question on the link:https://groups.google.com/forum/#!forum/jenkinsci-users[Jenkins users mailing list]
+* Answer a question on link:https://www.reddit.com/r/jenkinsci/[reddit]
```

Comment:
I suggest including the `jenkins` tag on Stack Overflow as well, probably above the subreddit